### PR TITLE
The invite user form now prompts for a worker's agency affiliations

### PR DESCRIPTION
**Pull request by sandlerben**

### CODE REVIEW

**Comment by rileynwong** [comment]
looks fine to me

File: app/templates/account/manage.html [hunk 1]
```diff
@@ -26,8 +26,8 @@
         <tr><td>Email address</td><td>{{ user.email }}</td></tr>
         <tr><td>Phone number</td><td>{{ user.phone_number }}</td></tr>
         <tr><td>Account type</td><td>{{ user.role.name }}</td></tr>
-        {% if user.agencies|length > 0 %}
-            <tr><td>Agency affiliations</td><td>{{ user.agencies|join(',', attribute='name') }}</td></tr>
+        {% if user.is_worker() %}
+            <tr><td>Agency affiliations</td><td>{{ user.agencies|join(', ', attribute='name') }}</td></tr>
         {% endif %}
```
Comment (kyle-rosenbluth) [inline app/templates/account/manage.html:31]: Can you explain the change here? Mainly, not sure what the `|` does in the old code.

**Comment by kyle-rosenbluth** [comment]
Merge at will :+1:

File: app/templates/account/manage.html (hunk 1 above)
Comment (sandlerben) [inline app/templates/account/manage.html:31]: @kyle-rosenbluth that's Jinja syntax for calling the length function.

**Pull request merged**